addons/mail/static/src/core/common/message.js

```diff
@@ -307,6 +307,10 @@ export class Message extends Component {
         return Boolean(this.env.inChatWindow && this.props.message.isSelfAuthored);
     }
 
+    get isNotOriginThreadAndIsNotTransient() {
+        return !this.isOriginThread && this.props.hasActions;
+    }
+
```

Comment:
`AndIsNotTransient` comes from previous code that was `and !message.is_transient` but this has been replaced with `this.props.hasActions`. Is it intentional?

`hasActions` is an option to show actions (when available) on the message. It doesn't mean that the message has no actions (e.g. as a consequence of being transient). For example, the Dialog shown to confirm message deletion shows the same message without the actions. That's one use-case of `props.hasActions = false`.

I would name the getter `isPersistentMessageFromAnotherThread`: it's easier to read when there's no `"Not"` (twice)